docs/markup-expr.md

```diff
@@ -3,5 +3,85 @@ id: markup-expr
 title: Expressions
 ---
 
-**TBD**
+Ergo Logic
+---
+
+In order to make a clause executable, there needs to be logic to act on the information linked between the template model to the natural language text that describes the legally enforceable clause. By defining computable logic based on the bindings between the template model and natural language text for the clause  via the Accord Project Ergo language, we create what we call the template logic. This determines how a syntactically valid clause performs.
+
+Computed Expressions 
+---
+
+Computed expressions are like formulas, support generation of the document text, and require [Ergo](https://docs.accordproject.org/docs/logic-ergo.html) knowledge.
+
+Expressions are quoted with `{{% ... %}}`
+
+Many special purpose things in handlebars can be written as Ergo expressions:
+
+### Nested Paths
+
+Computing the paths to values within an object:
+
+```handlebars
+{{% author.name %}}
+```
+
+### Helpers
+
+Function calls as helpers:
+
+```handlebars
+{{% roundn(rate,precision) %}}
+```
+
+### Literals
+
+Function calls as literals:
+
+```handlebars
+{{% roundn(rate,2) %}}
+```
+
+### Iterators
+
+Loop through items in the model inline the document:
+
+```handlebars
+{{% foreach p in products
+    return `Product: {{ p.name }}` %}}
+```
+
+### Conditionals
+
+Evaluation of an `if...else` statement inline:
+
+```handlebars
+{{% if state = "US"
+    then address.state
+    else address.country %}}
+```
+
+### String Concatenation
+
+Computing the value of string concatenation inline:
+
+```handlebars
+{{% firstName ++ lastName %}}
+```
+
+Currently Unsupported
```

Comment:
I would remove this. We don't want to confuse the reader or advertise features we may or may not support in the future. This is fine to leave those as GitHub issues.